### PR TITLE
React router implemented with basic routes

### PR DESCRIPTION
**Pull request by Cililocwee**
The routes for the main pages are there. I extracted a bunch of extra code used for development that are unnecessary in the current build. The auth's log in / sign out button is still present but should be removed one the login/signout function is hooked up in the navbar.

### CODE REVIEW

**Comment by Cililocwee** [comment]
And the navbar needs to be extracted as a separate component and embedded above the rendering area to persist across pages (separate and higher up in the Router component)

**Comment by Cililocwee** [comment]
After conferring with Tim, I added further routes

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
+            />
+            <Route path="/terms" element={<h1>Terms Page</h1>} />
+            <Route
+              path="/privacy-policy"
+              element={<h1>Privacy Policy Page</h1>}
+            />
+            <Route path="/profile" element={<h1>Profile Page</h1>} />
+            <Route path="/login" element={<h1>Log In Page</h1>} />
+            <Route path="/signup" element={<h1>Sign Up Page</h1>} />
+            <Route path="/*" element={<h1>404: Page not found</h1>} />
```
Comment (jtabba) [inline frontend/src/App.tsx]: Great attention to detail 👏

File: frontend/src/components/NavBar.tsx [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+const NavBar: FC<NavBarProps> = ({ isLoggedIn }) => {
+  return (
+    <Flex
+      justifyContent="space-between"
+      alignItems="center"
+      h="60px"
+      outline={1}
+    >
+      <Logo />
+      {/*isLoggedIn ? <NavLinks /> : null */}
```
Comment (jtabba) [inline frontend/src/components/NavBar.tsx]: If we don't need it lets remove it

**Review by jtabba — APPROVED** [review]

**Pull request merged**